tools/pkglint/lib/rules.ts

```diff
@@ -1457,6 +1457,43 @@ export class JestSetup extends ValidationRule {
   }
 }
 
+/**
+ * Most packages in v2 must be private. Carry an allowlist so we don't
+ * accidentally publish private packages.
+ */
+export class V2PublicPackages extends ValidationRule {
+  public readonly name = 'v2/public-package';
+  private readonly allowlist = ['aws-cdk-lib', 'cdk', 'aws-cdk', 'awslint'];
+
+  public validate(pkg: PackageJson): void {
+    // eslint-disable-next-line @typescript-eslint/no-require-imports
+    const releaseJson = require(`${__dirname}/../../release.json`);
```

Comment:
As the build is pointing out, this path isn't necessarily correct for each module, including our tools.

`Error: Cannot find module '/codebuild/output/src507724372/src/github.com/aws/aws-cdk/tools/pkglint/lib/../../release.json'`